test/requirements.txt

```diff
@@ -1,31 +1,31 @@
-amqp==1.4.8
-ansible==1.8.4
-anyjson==0.3.3
-flake8==3.2.1
-gevent==1.1.2
-greenlet==0.4.10
-jsonmerge==1.2.1
-jsonschema==2.4.0
-kombu==3.0.30
-nose==1.3.7
-nosedep>=0.6
-on-http_api1_1>=1.0.0
-on-http_api2_0>=1.0.2
-on_http_redfish_1_0>=1.0.0
-pexpect==3.3
-proboscis==1.2.6.0
-requests==2.9.0
-# there are problems with the 34.0.x setuptools and nosedeps. Pin for now.
-setuptools==33.1.1
-# urllib3 1.20 breaks CIT test, specify version as a temp solution
-urllib3==1.19.1
-
-# WARNING: do not just replace this file with "pip freeze > requirements.txt".
-# You need to keep the following line as is! (freeze will turn it into
-# a https/git reference)
--e fit_path
--e stream-monitor
-
-# WARNING: there is an ubuntu bug that causes a bad package to get listed when doing a freeze.
-#  Do NOT add 'package-resources==0.0.0' if you see it! I also see 'wheel=nnnnnn' showing up at times.
-#  That's kind of infra also, so don't add it either!
+amqp==1.4.8
+ansible==1.8.4
+anyjson==0.3.3
+flake8==3.2.1
+gevent==1.1.2
+greenlet==0.4.10
+jsonmerge==1.2.1
+jsonschema==2.4.0
+kombu==3.0.30
+pika=0.10.0
+nose==1.3.7
+nosedep>=0.6
+on-http_api1_1>=1.0.0
+on-http_api2_0>=1.0.2
+on_http_redfish_1_0>=1.0.0
+pexpect==3.3
+proboscis==1.2.6.0
+requests==2.9.0
+# there are problems with the 34.0.x setuptools and nosedeps. Pin for now.
+setuptools==33.1.1
+# urllib3 1.20 breaks CIT test, specify version as a temp solution
+urllib3==1.19.1
+
+# WARNING: do not just replace this file with "pip freeze > requirements.txt".
+# You need to keep the following line as is! (freeze will turn it into
+# a https/git reference)
+-e stream-monitor
+
+# WARNING: there is an ubuntu bug that causes a bad package to get listed when doing a freeze.
+#  Do NOT add 'package-resources==0.0.0' if you see it! I also see 'wheel=nnnnnn' showing up at times.
+#  That's kind of infra also, so don't add it either!

```

Comment:
please don't add c/r's to files. It makes it difficult to see what actually changed.